e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -88,7 +89,20 @@ context('When fee record correction feature flag is enabled', () => {
 
       cy.clickContinueButton();
 
-      cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}/check-the-information`));
+      //---------------------------------------------------------------
+      // "Check the information" page
+      //---------------------------------------------------------------
+      cy.assertText(mainHeading(), 'Check the information before submitting the record correction request');
+
+      summaryList().should('exist');
+      summaryList().should('contain', feeRecordAtToDoStatus.facilityId);
```

Comment:
Can any of these use cy.assertText?

Reply:
I can't add data-cy directly to the items so can't access them directly via a data-cy. I could get the exact element and then use cy.assertText, by using a complex html selector but those selectors would be quite brittle, so I thought it would be best to have a slightly less strict assertion, but if you would prefer it that way I can change it.